database_lookup_search.py

```diff
@@ -12,6 +12,26 @@
 padx = 10
 pady = 5
 
+def load_all_patients():
```

Comment:
Loading all data like this will get bad as data amount goes up, should probably just query this data from the database instead of loading it into memory again.